docs/source/developer_guide/core/caching.rst

```diff
@@ -23,11 +23,19 @@ Below are some methods you can use to control how the hashes of calculation and
 Controlling caching
 -------------------
 
-There are two methods you can use to disable caching for particular nodes:
+There are several methods you can use to disable caching for particular nodes:
+
+On the level of generic :class:`aiida.orm.nodes.Node`:
 
 * The :meth:`~aiida.orm.nodes.Node.is_valid_cache` property determines whether a particular node can be used as a cache. This is used for example to disable caching from failed calculations.
 * Node classes have a ``_cachable`` attribute, which can be set to ``False`` to completely switch off caching for nodes of that class. This avoids performing queries for the hash altogether.
 
+On the level of :class:`aiida.engine.processes.process.Process` and :class:`aiida.orm.nodes.process.ProcessNode`:
+
+* The :meth:`ProcessNode.is_valid_cache <aiida.orm.nodes.process.ProcessNode.is_valid_cache>` calls :meth:`Process.is_valid_cache <aiida.engine.processes.process.Process.is_valid_cache>`, passing the node itself. This can be used in :class:`~aiida.engine.processes.process.Process` subclasses (e.g. in calculation plugins) to implement custom ways of invalidating the cache.
```

Comment:
you can shorthen these `:meth:ProcessNode.is_valid_cache <aiida.orm.nodes.process.ProcessNode.is_valid_cache>` to `:meth:~aiida.orm.nodes.process.ProcessNode.is_valid_cache` which is what we use elsewhere in the documentation

Reply:
Tried that initially, but then it just writes "``is_valid_cache()`` calls `is_valid_cache()`` (i.e., the class name is dropped completely. In this case, that doesn't convey the actual meaning.

Reply:
I see, fine to keep as is then